### ISSUE TITLE
Unable to install through pip 

### ISSUE THREAD

**Issue by PSSF23** [scikit-garden/scikit-garden#103]
`pip` installation doesn't recognize already installed `numpy`.
```
Collecting scikit-garden
  Downloading scikit-garden-0.1.3.tar.gz (317 kB)
     |████████████████████████████████| 317 kB 78.6 MB/s            
  Preparing metadata (setup.py) ... - error
  ERROR: Command errored out with exit status 1:
   command: /home/circleci/project/venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_2d8007f33a52439ab401c4ec290abf09/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_2d8007f33a52439ab401c4ec290abf09/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-92p854xv
       cwd: /tmp/pip-install-xf6kw9yr/scikit-garden_2d8007f33a52439ab401c4ec290abf09/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-xf6kw9yr/scikit-garden_2d8007f33a52439ab401c4ec290abf09/setup.py", line 6, in <module>
      import numpy
  ModuleNotFoundError: No module named 'numpy'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/63/0f/850592f6e8619d42086cdcc6a1731a4254f74db09878a155ab3d4cdc55ef/scikit-garden-0.1.3.tar.gz#sha256=d6fe65205f39633f17c181f552a0fc56e2f05bb2096be63f38c160fe20718d9d (from https://pypi.org/simple/scikit-garden/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  Downloading scikit-garden-0.1.2.tar.gz (313 kB)
     |████████████████████████████████| 313 kB 83.0 MB/s            
  Preparing metadata (setup.py) ... - error
  ERROR: Command errored out with exit status 1:
   command: /home/circleci/project/venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_806822fed9e24f09a5555aebd9c5be89/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_806822fed9e24f09a5555aebd9c5be89/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-qu8tg4u8
       cwd: /tmp/pip-install-xf6kw9yr/scikit-garden_806822fed9e24f09a5555aebd9c5be89/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-xf6kw9yr/scikit-garden_806822fed9e24f09a5555aebd9c5be89/setup.py", line 6, in <module>
      import numpy
  ModuleNotFoundError: No module named 'numpy'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/e8/f7/38a1107426e7a9b0a2bc1a35f5325a674b5e2a804d8e67f6a20846bcb85c/scikit-garden-0.1.2.tar.gz#sha256=1ac32c80e33bcb65aa02696e98c005e5bea51acef6b012ce4153a3e9b5838ec9 (from https://pypi.org/simple/scikit-garden/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  Downloading scikit-garden-0.1.1.tar.gz (317 kB)
     |████████████████████████████████| 317 kB 96.8 MB/s            
  Preparing metadata (setup.py) ... - error
  ERROR: Command errored out with exit status 1:
   command: /home/circleci/project/venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_c3cf1275d496488fbbdb2a926248c87a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_c3cf1275d496488fbbdb2a926248c87a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-3ixzhpoe
       cwd: /tmp/pip-install-xf6kw9yr/scikit-garden_c3cf1275d496488fbbdb2a926248c87a/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-xf6kw9yr/scikit-garden_c3cf1275d496488fbbdb2a926248c87a/setup.py", line 6, in <module>
      import numpy
  ModuleNotFoundError: No module named 'numpy'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/e2/ed/00a1029cc08c7abcd75ece54dce1b1df3bed7544b507a2512199d5c7fba4/scikit-garden-0.1.1.tar.gz#sha256=6d5c7749c39a62b19eb5b8335ea7594412d7bb37a287272363855158c6ccb2b9 (from https://pypi.org/simple/scikit-garden/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  Downloading scikit-garden-0.1.tar.gz (299 kB)
     |████████████████████████████████| 299 kB 82.6 MB/s            
  Preparing metadata (setup.py) ... - error
  ERROR: Command errored out with exit status 1:
   command: /home/circleci/project/venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_d39a62ffa5e340919b139872e4b8046a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-xf6kw9yr/scikit-garden_d39a62ffa5e340919b139872e4b8046a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-csnck_bh
       cwd: /tmp/pip-install-xf6kw9yr/scikit-garden_d39a62ffa5e340919b139872e4b8046a/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-xf6kw9yr/scikit-garden_d39a62ffa5e340919b139872e4b8046a/setup.py", line 6, in <module>
      import numpy
  ModuleNotFoundError: No module named 'numpy'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/28/aa/b58d5bcd024b9b7f9737ab319c4da248154d33611e1b4167274e5f942028/scikit-garden-0.1.tar.gz#sha256=29218a2fb7687c897b2dc65032265eb7c95b267b26e1d2a40c3b29477bb326a5 (from https://pypi.org/simple/scikit-garden/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement scikit-garden (from versions: 0.1, 0.1.1, 0.1.2, 0.1.3)
ERROR: No matching distribution found for scikit-garden

Exited with code exit status 1
CircleCI received exit code 1
```

**Comment by PSSF23** [comment]
Also, the current version on `pip` is not up-to-date (not including changes from #89).

**Comment by ecederstrand** [comment]
This was fixed in #86
